src/components/calendar/Calendar.tsx

```diff
@@ -11,6 +11,18 @@ import * as React from 'react';
 import * as _ from 'underscore';
 import * as moment from 'moment';
 
+export const CalendarSelectionRuleType = {
+  all: 'ALL',
+  lower: 'LOWER',
+  upper: 'UPPER',
+  range: 'RANGE'
+};
+
+export interface ICalendarSelectionRule {
+  test: (date: Date, secondDate?: Date) => boolean;
+  isFor: 'ALL' | 'LOWER' | 'UPPER' | 'RANGE';
```

Comment:
In the d.ts, is it typed as string? Because I install react-vapor in UA and it will probably break since i'm not in typescript 2